WordPress/wordpress-playground
# Add PDF to infer mime type list

## What is this PR doing?

Enables static PDF files to be returned. 

## What problem is it solving?

It will allow the download of static PDF files.

## How is the problem addressed?

By adding PDF to the infer mime type list. 

## Testing Instructions

- Check out this branch
- [Open Playground ](http://127.0.0.1:5400/website-server/?url=/wp-admin/upload.php)
- Upload a PDF as media
- Right click on Download file an open the link in a new tab
- See that the PDF loads
